src/smk/Window.cpp

```diff
@@ -107,6 +107,11 @@ void GLFWCharCallback(GLFWwindow* glfw_window, unsigned int codepoint) {
 
 }  // namespace
 
+glm::vec2 smk::Window::ToScreenSpace(const glm::vec2& world) {
+  glm::vec2 factor = glm::vec2(initialSize_.x/width_, initialSize_.y/height_);
+  return glm::vec2(world.x*factor.x, world.y*factor.y);
```

Comment:
The view can also be rotated / skewed.

https://github.com/ArthurSonzogni/smk/blob/24049a5caee8753d2414a258de1fd23487c0a98b/src/smk/RenderTarget.cpp#L74

- I think it would be better to use the RenderTarget::project_matrix for this.
- I don't think we need to keep track of the initial size.
- I think this can be moved to RenderTarget.

Reply:
Hm yeah the view could be accounted for. That would make sense and become very useful. My original intention was to match the window size even after resizing. I originally had this event handled by `Input` and used the resize event for glfw to feed it, but I thought maybe it was more appropriate for the window.